apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -1166,34 +1166,49 @@ export function editedSectionId(state) {
 }
 
 /**
+ * @param {object} state - state.teacherSections in redux tree
  * Extract a list of name/id for each section
  */
 export function sectionsNameAndId(state) {
-  return state.sectionIds.map(id => ({
-    id: parseInt(id, 10),
-    name: state.sections[id].name
-  }));
+  return sortSectionsList(
+    state.sectionIds.map(id => ({
+      id: parseInt(id, 10),
+      name: state.sections[id].name
+    }))
+  );
 }
 
+/**
+ * @param {object} state - state.teacherSections in redux tree
+ */
 export function sectionsForDropdown(
   state,
   scriptId,
   courseId,
   onCourseOverview
 ) {
-  return state.sectionIds.map(id => ({
-    id: parseInt(id, 10),
-    name: state.sections[id].name,
-    scriptId: state.sections[id].scriptId,
-    courseId: state.sections[id].courseId,
+  return sortedSectionsList(state.sections).map(s => ({
+    ...s,
```

Comment:
small style thought - we tend to use full words rather than single-letter variable names when possible to make the code more readable, so consider changing `s` to `section`

Reply:
👍